src/js/apps/forms/form/form-patient_app.js

```diff
@@ -1,5 +1,7 @@
 import Radio from 'backbone.radio';
 import dayjs from 'dayjs';
+import store from 'store';
+import { extend } from 'underscore';
```

Comment:
super nitpick, but I think the underscore import should be first above Radio

Reply:
10-4, I'll get that updated.